src/main/kotlin/io/github/mojira/arisa/modules/EmptyModule.kt

```diff
@@ -37,8 +37,8 @@ class EmptyModule(
                 assertNotEqual(environment, ENV_DEFAULT).bind()
             }
             assertEmpty(attachments).bind()
-            addComment(CommentOptions(message)).toFailedModuleEither().bind()
-            resolveAsIncomplete().toFailedModuleEither().bind()
+            addComment(CommentOptions(message))
```

Comment:
add comment should be always after resolving, and it should only happen if the module didnt fail to do the resolve